src/components/relaying-education-series/IndexNumber/index.tsx

```diff
@@ -0,0 +1,15 @@
+import { Typography } from '@mui/material'
+import css from './styles.module.css'
+
+const IndexNumber = ({ value }: { value: number }) => {
+  return (
+    <Typography className={css.step}>
+      {value.toLocaleString('en-US', {
```

Comment:
This should be localised.

```suggestion
      {value.toLocaleString(undefined, {
```

Reply:
oops 👍

Reply:
![](https://media4.giphy.com/media/QmKHRjIwe4M24V3VcM/giphy.gif)